.azure-pipelines/azure-pipelines-linux.yml

```diff
@@ -35,9 +40,11 @@ jobs:
   - script: |
         export CI=azure
         export GIT_BRANCH=$BUILD_SOURCEBRANCHNAME
+        export FEEDSTOCK_NAME=$(basename ${BUILD_REPOSITORY_NAME})
+        echo "git branch: ${GIT_BRANCH}"
```

Comment:
@isuruf code I executed is here

Reply:
I guess we need `System.PullRequest.TargetBranch`

Reply:
I’m gonna document this all in the GitHub issue for ppl to work on if they want.

Reply:
The docs say "this variable is only available in a YAML pipeline if the PR is affected by a branch policy." I am thinking it may not be there for our builds. The branch policies are set in azure and AFAIK we don't use any. 

@mariusvniekerk do you happen to remember if we set any azure branch policies?

Reply:
AHHH NVM. This appears to work. IDK what that comment in their docs means.

Reply:
We want the target branch FWIW on PRs. Not sure about master.